app/src/state/editor/types.ts

```diff
@@ -46,9 +46,11 @@ export interface EditorState {
   displayVideo: boolean;
 
   exportState: ExportState;
-  exportPopup: boolean;
+  exportPopup: ExportPopupState;
 }
 
+export type ExportPopupState = false | 'document' | 'selection';
```

Comment:
Nit: I don't like mixing a string and a boolean here. Maybe `'hidden'` or something similar instead?